src/org/zaproxy/zap/extension/anticsrf/AntiCsrfDetectScanner.java

```diff
@@ -86,4 +87,20 @@ public void setLevel(AlertThreshold level) {
 		// Ignore
 	}
 
+	private boolean tagHistoryType(int historyType) {
```

Comment:
This code is repeated in a bunch of places, both as `tagHistoryType` and `appliesToHistoryType`, perhaps it could be centralized?

Or perhaps a constant definition of the applicable history types could be added to HistoryReference and then referenced in all these places, that way if it needs maintenance down the road it's only a single location that has to be modified. (Maybe within the zap.utils.Stats or zap.utils.Statistics ... would be a better home than HistoryReference?)